PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawComparedGraphs.java

```diff
@@ -0,0 +1,308 @@
+package nl.tudelft.pl2016gr2.gui.view.graph;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollBar;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.Region;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Circle;
+import javafx.scene.shape.Line;
+import nl.tudelft.pl2016gr2.model.AbstractNode;
+import nl.tudelft.pl2016gr2.model.GraphNodeOrder;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.ResourceBundle;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Draws two compared graphs above each other in a pane.
+ *
+ * @author Faris
+ */
+public class DrawComparedGraphs implements Initializable {
```

Comment:
Well this is a controller class. Fxml/css files are basically the pure visual representation components, the controller class of a fxml file decides what is actually shown in every part of the view (only the objects, usually not what they look like) and what actions are performed when the user interacts with objects in the view. There are however exceptions to this (for example if the shade of a color depends on a certain value).
The class basically only draws the tree, so this is pure GUI stuff. The only potential exception to this could be the `repositionOverlappingEdges` method, which shifts nodes up if inlinks are overlapping, but this also is a visual thing, so in my opinion belongs in this class.